### PR TITLE
fix for #37

### PR DESCRIPTION
**Pull request by tarjei**
Hi, 
This should fix issue #37 - create an operation to split a file
with multiple classes into separate files.

The next thing now is to create an operation to add namespaces and
move files into the correct directory based on that namespace.

### CODE REVIEW

File: ScisrRunner.php [hunk 1]
```diff
@@ -270,6 +270,26 @@ class ScisrRunner
...
+     * @param string $oldFilePath the path to the file to be split
+     * @param string $outputPath the path where the files should be saved
+     */
+    public function setSplitClassFiles($outputPath)
+    {
+        $this->_listeners[] = $this->_operationsFactory->getOperation('Scisr_Operations_SplitClassFile', $outputPath);
+        if (!is_dir($outputPath) || !is_writeable($outputPath)) {
+            $msg = $outputPath . ' is not writeable or a directory.';
+        } else {
+            //$this->_changeRegistry->addSplit($oldFilePath, $outputPath);
```
Comment (iangreenleaf) [inline ScisrRunner.php:284]: Left over by accident?

File: .gitignore [hunk 2]
```diff
@@ -1 +1,2 @@
 cache.db
+.*.swp
```
Comment (iangreenleaf) [inline .gitignore:2]: I prefer putting system-specific rules in a global gitignore rather than my projects' ignore lists.

**Comment by iangreenleaf** [comment]
Awesome, this looks very nice. Thanks!

File: ScisrRunner.php (hunk 1 above)
Comment (tarjei) [inline ScisrRunner.php:284]: Yep. Feel free to remove it :)

**Pull request merged**